File: src/Pidev/FrontBundle/Entity/OffreMission.php
<?php

namespace Pidev\FrontBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * OffreMission
 *
 * @ORM\Table(name="offre_mission")
 * @ORM\Entity
 */
class OffreMission
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id_offre", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idOffre;

    /**
     * @var string
     *
     * @ORM\Column(name="lieu_mission", type="string", length=25, nullable=false)
     */
    private $lieuMission;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_debut", type="date", nullable=false)
     */
    private $dateDebut;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_fin", type="date", nullable=false)
     */
    private $dateFin;

    /**
     * @var string
     *
     * @ORM\Column(name="horaire_garde", type="string", length=1000, nullable=false)
     */
    private $horaireGarde;

    /**
     * @var integer
     *
     * @ORM\Column(name="nombre_enfant", type="integer", nullable=false)
     */
    private $nombreEnfant;

    /**
     * @var string
     *
     * @ORM\Column(name="sexe", type="string", length=20, nullable=false)
     */
    private $sexe;

    /**
     * @var integer
     *
     * @ORM\Column(name="id_parent", type="integer", nullable=true)
     */
    private $idParent;

    /**
     * @var integer
     *
     * @ORM\Column(name="id_service", type="integer", nullable=true)
     */
    private $idService;
    function getIdOffre() {
        return $this->idOffre;
    }

    function getLieuMission() {
        return $this->lieuMission;
    }

    function getDateDebut() {
        return $this->dateDebut;
    }

    function getDateFin() {
        return $this->dateFin;
    }

    function getHoraireGarde() {
        return $this->horaireGarde;
    }

    function getNombreEnfant() {
        return $this->nombreEnfant;
    }

    function getSexe() {
        return $this->sexe;
    }

    function getIdParent() {
        return $this->idParent;
    }

    function getIdService() {
        return $this->idService;
    }

    function setIdOffre($idOffre) {
        $this->idOffre = $idOffre;
    }

    function setLieuMission($lieuMission) {
        $this->lieuMission = $lieuMission;
    }

    function setDateDebut(\DateTime $dateDebut) {
        $this->dateDebut = $dateDebut;
    }

    function setDateFin(\DateTime $dateFin) {
        $this->dateFin = $dateFin;
    }

    function setHoraireGarde($horaireGarde) {
        $this->horaireGarde = $horaireGarde;
    }

    function setNombreEnfant($nombreEnfant) {
        $this->nombreEnfant = $nombreEnfant;
    }

    function setSexe($sexe) {
        $this->sexe = $sexe;
    }

    function setIdParent($idParent) {
        $this->idParent = $idParent;
    }

    function setIdService($idService) {
        $this->idService = $idService;
    }



}

File: src/edu/esprit/dao/entities/OffreMission.java


package edu.esprit.dao.entities;

import java.sql.Date;


public class OffreMission {
    
    private int idOffre ;
    private String lieuMission; 
    private Date dateDebut ;
    private Date dateFin ;
    private String HoraireDeGarde ;
    private int nombreEnfants;
    private String typeService ;
    

 public OffreMission(int idOffre,String lieuMission,Date dateDebut, Date DateFin,String horaireGarde , int nb ) {
        this.idOffre=idOffre ;
        this.lieuMission=lieuMission ;
        this.dateDebut=dateDebut ;
        this.dateFin=DateFin ;
       this.HoraireDeGarde=horaireGarde;
       
       this.nombreEnfants=nb;
       
       
       
    }

    public OffreMission() {
       
    }

    
    public int getIdOffre() {
        return idOffre;
    }

    
    public void setIdOffre(int idOffre) {
        this.idOffre = idOffre;
    }

    
    public String getLieuMission() {
        return lieuMission ;
    }

    
  

    /**
     * @return the dateDebut
     */
    public Date getDateDebut() {
        return dateDebut;
    }

    /**
     * @param dateDebut the dateDebut to set
     */
    public void setDateDebut(Date dateDebut) {
        this.dateDebut = dateDebut;
    }

    /**
     * @return the datefin
     */
    public Date getDateFin() {
        return dateFin;
    }

    /**
     * @param datefin the datefin to set
     */
    public void setDateFin(Date Datefin) {
        this.dateFin = Datefin;
    }

    public   void afficher()
    {
        System.out.println("ID : "+idOffre+"\nLieu de mission : "+lieuMission+"\nDate de Debut : "+dateDebut+"\nDate de Fin : "+dateFin);
    }

    
    public String getHoraireDeGarde() {
        return HoraireDeGarde;
    }

    /**
     * @return the nombreEnfants
     */
    public int getNombreEnfants() {
        return nombreEnfants;
    }

    

    /**
     * @param lieuMission the lieuMission to set
     */
    public void setLieuMission(String lieuMission) {
        this.lieuMission = lieuMission;
    }

    /**
     * @param HoraireDeGarde the HoraireDeGarde to set
     */
    public void setHoraireDeGarde(String HoraireDeGarde) {
        this.HoraireDeGarde = HoraireDeGarde;
    }

    /**
     * @param nombreEnfants the nombreEnfants to set
     */
    public void setNombreEnfants(int nombreEnfants) {
        this.nombreEnfants = nombreEnfants;
    }

   
   
}
